File: 1550. Three Consecutive Odds/1550.myans.go
func threeConsecutiveOdds(arr []int) bool {
    count := 0
    for _, x := range arr {
        if x % 2 == 1 {
            count += 1
            if count == 3 { return true }
        } else {
            count = 0
        }
    }
    return false 
}

File: 1550. Three Consecutive Odds/1550.myans.rs
impl Solution {
    pub fn three_consecutive_odds(arr: Vec<i32>) -> bool {
        let mut count = 0;
        for &x in &arr {
            if x % 2 == 1 {
                count += 1;
                if count == 3 { return true; }
            } else {
                count = 0;
            }
        }
        false
    }
}